client/src/components/MapView.vue

```diff
@@ -123,6 +123,29 @@ export default defineComponent({
       });
     },
 
+    /**
+     * Set up map controls.
+     *
+     * This currently consists of the GeolocateControl which sets up
+     * functionality to pan to the user's current location.
+     */
+    setUpControls(): void {
+      if (this.map == null) return;
+
+      // Add geolocate control to the map.a
+      this.map.addControl(
```

Comment:
Don't feel like you need to address in this PR, but does this work on mobile?

Reply:
Not exactly sure from documentation, it just says that 'not all browsers support geolocation'. For mobile, it will still be opened in a web browser so I think it should be the same as whether that browser is supported in general? Not exactly sure though.